common/utils/ServiceFlowMapper.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const _ = require('lodash');
+const CONST = require('../constants');
+const logger = require('../logger');
+
+class ServiceFlowMapper {
+  getServiceFlow(params) {
+    let serviceFlow;
+    const flowCheckers = ['_checkForMultiAz', '_checkForBluePrint'];
```

Comment:
No. These are function names within the same class. They are not resuable. Conveting them to consts would be overdoing and wont achieve the benefits of having constants truely. Also it would affect readability badly in this case.